curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/66605a64a9822f01ec5ae8ce.md

```diff
@@ -0,0 +1,50 @@
+---
+id: 66605a64a9822f01ec5ae8ce
+title: Step 50
+challengeType: 20
+dashedName: step-50
+---
+
+# --description--
+
+In Python, the scope of a variable determines where that variable can be accessed. Variables defined inside a function are local to that function and cannot be accessed outside of it.
+
+Let's see this in action. At the end of your code, try to print the `alphabet` variable.
```

Comment:
```suggestion
To see this in action, try to print the `alphabet` variable at the end of your code. This will raise a `NameError` exception.
```
We agreed not to use "let's" and "we".